tests/hipscat_import/margin_cache/test_margin_cache.py

```diff
@@ -37,6 +38,10 @@ def test_margin_cache_gen(small_sky_source_catalog, tmp_path, dask_client):
 
     assert len(data) == 13
 
+    assert all(data[PartitionInfo.METADATA_ORDER_COLUMN_NAME] == norder)
```

Comment:
Can you please add assertions on the data types of the partition info columns? see also: https://github.com/astronomy-commons/hipscat-import/issues/161

Should be uint8 for Norder and uint64 for Dir and Npix.

Reply:
Got it, updated to convert to and assert those